anyio/__init__.py

```diff
@@ -501,7 +501,8 @@ async def create_unix_server(
 
 async def create_udp_socket(
     *, interface: Optional[IPAddressType] = None, port: Optional[int] = None,
-    target_host: Optional[IPAddressType] = None, target_port: Optional[int] = None
+    target_host: Optional[IPAddressType] = None, target_port: Optional[int] = None,
+    address_family: Optional[int] = socket.AF_UNSPEC
```

Comment:
What would the address family be for multicast?

Reply:
Never mind the previous question, but shouldn't the default be something more sensible, like `AF_INET`?

Reply:
Also, wouldn't you specify a target host/port when creating a socket for multicast?

Reply:
Well, there's AF_INET6 …
You can't specify a target because `connect()`ing to a UDB socket means that you only accept messages from that address. Multicast packets however cannot have a multicast source.